src/decentralized-lending.yaml

```
@@ -12,24 +12,476 @@ publishStatus: Live
+      
+      ### Important Terms 
+      
+      1. **APY**-The annual percentage yield (APY) is the real rate of return earned on an investment, taking into account the effect of compounding interest. 
+      2. **atokens**- atokens are the interest-bearing tokens which is minted and burnt in 1:1 ratio upon deposit and redeem.
+      3. **ERC-20-** The ERC-20 introduces a standard for Fungible Tokens. In other words, they have a property that makes each Token precisely the same 
+      (in type and value) as another Token.
+      4. **Deposit interest**-A deposit interest rate is the rate at which you earn money from a bank or financial institution on your deposits in an 
+      interest-bearing account.
+      5. **Stablecoins**-A stablecoin is a digital currency pegged to a stable reserve asset like the U.S. dollar or gold. Stablecoins are designed to 
+      reduce volatility relative to unpegged cryptocurrencies like Bitcoin.
+      6. **Flashloan -**Loans can be borrowed without collateral through smart contracts. This process is instant and can help borrowers get the money 
+      they need quickly.
+      7. **ROI - **Return on investment is a performance measure used to evaluate the efficiency or profitability of an investment or compare the 
+      efficiency of several different investments.
+      8. **LTV-** loan to value, it represents the borrowing power of the collateral.
+      9. **Health Factor -** The health factor determines whether a user is eligible to borrow a loan or not.
     stepItems: []
-  - uuid: 3aac50b3-ac05-4add-88df-4d94f6fb5ae0
-    name: How to deposit money in AAVE
+  - uuid: 6511e9c8-3215-4629-800b-aed2b8b37ef4
+    name: Evaluation
+    content: |
+    stepItems:
+      - uuid: fe3d165c-c1fb-4ca2-a62d-12ba331a6955
+        content: What are atokens?
+        answerKeys:
+          - B
+        type: SingleChoice
+        choices:
+          - content: atokens are stablecoins like USDT
+            key: A
+          - content: atokens are interest bearing tokens which is minted and burnt in 1:1 ratio upon deposit and redeem.
+            key: B
+          - content: atokens are tokens which give governance in Aave
+            key: C
+          - content: None of these
+            key: D
+
+      - uuid: 5d9dfe2a-f030-4f6e-8e9c-895768a7d316
+        content: What are stablecoins?
+        answerKeys:
+          - A
+        type: MultipleChoice
+        choices:
+          - content: A stablecoin is a digital currency pegged to a stable reserve asset like the U.S. dollar or gold.
+            key: A
+          - content: stablecoins are an allternate coin for Bitcoin.
+            key: B
+          - content: stable coins are more volatile coins
+            key: C
+          - content: None of these
+            key: D
+            
+      - uuid: 97bd3ac5-e4f1-45ef-a40f-6e601020b090
+        content: Which of the following are correct about lending in crypto?
```

```suggestion
        content: Which of the following are correct about lending in AAVE?
```

Please change crypto to AAVE in the choices too